### ISSUE TITLE
Status effects

### ISSUE THREAD

**Issue by findusl** [findusl/initiative-tracker#121]
Add support for status effects.

- support duration
- decrease duration on new round to avoid accidental decrease when jumping
- show special icon when decreasing
- show dialog when decreasing to warn user with option to not show again
- show Snackbar when decreasing
- show status effects in list view for everyone 
- provide option to increase status effect for all if accidentally decreasef
- add option to remove status effect (e.g. When saved)
- stretch goal: allow managing status effect per creature
- stretch goal: allow managing all active status effects and group if same for different creatures
- stretch goal: automatically calculate save on turn end. Give Snackbar warning. Maybe ensure only once per round
- round counter probably necessary
